src/Infrastructure.Dapper/AdminConsole/Repositories/OrganizationUserRepository.cs

```diff
@@ -329,7 +329,7 @@ public async Task<Guid> CreateAsync(OrganizationUser obj, IEnumerable<Collection
         using (var connection = new SqlConnection(ConnectionString))
         {
             var results = await connection.ExecuteAsync(
-                $"[{Schema}].[OrganizationUser_CreateWithCollections_V2]",
+                $"[{Schema}].[OrganizationUser_CreateWithCollections]",
```

Comment:
The BRE team is looking to leverage EDD to be able to deploy database changes independent of code changes. The general pattern is to just drop old versions, but with renaming it here we are tying the database change with the code. @withinfocus Thoughts?

Reply:
Correct with your catch here. While more tedious, one change and deployment would be your proc duplication (both "V1" and "V2" now having the latter's content), then a second deployment would be the reference back to the V1 version, then finally a third being the V2 drop.